CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.1.1
+* Remove '_' prefix from generated structs and enums.
```

Comment:
There should be a blank line between the version name and the description by convention.